changelog.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [Unreleased]
+
+### Added
+
+- Add optional containerId argument to DefaultLayout.rearrangeComponentsPosition
```

Comment:
put DefaultLayout.rearrangeComponentsPosition in `` .
Missing `.` at the end.

Reply:
done